legend-engine-xts-authentication/legend-engine-xt-authentication-connection-factory/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Lets not call this module connection-factory.  Maybe just connection ?

Reply:
that's fair, I'll do it in future PR once we're set with the approach